utils/multiplatform-common/src/commonMain/kotlin/com/bumble/appyx/utils/multiplatform_common/Parcelable.kt

```diff
@@ -0,0 +1,11 @@
+package com.bumble.appyx.utils.multiplatform_common
```

Comment:
I'd avoid underscores in directory structure & package names (the module name is fine as is). Can we go with either two levels `multiplatform/common/` (and `multiplatform.common` respectively), or just a single one called `multiplatform`?